src/rviz_visual_tools.cpp

```diff
@@ -280,7 +280,9 @@ void RvizVisualTools::loadMarkerPub(bool wait_for_subscriber, bool latched)
   }
 
   // Rviz marker publisher
-  pub_rviz_markers_ = nh_.advertise<visualization_msgs::MarkerArray>(marker_topic_, 10, latched);
+  ros::NodeHandle nh(nh_.getNamespace());
```

Comment:
can you make this ``nh`` var name more descriptive, so its clear how its diff than ``nh_``?